core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/utils/TestStatusLogger.java

```diff
@@ -0,0 +1,194 @@
+        assertEquals("Total repair failures in node till now is: 1", logMessage);
+        assertEquals(Level.DEBUG, logLevel);
+        assertEquals(1, count);
+    }
+
+    @Test
+    public void testLogForSuccessfulRepairSessionCountLogs()
+    {
+        TableReference tableReference = tableReference(TEST_KEYSPACE, TEST_TABLE1);
+        long expectedRepairTime = 12345L;
+        myTableRepairMetricsImpl.repairSession(tableReference, expectedRepairTime, TimeUnit.MILLISECONDS, true);
+
+        StatusLogger.log(myMeterRegistry);
+
+
+        List<ILoggingEvent> logsList = listAppender.list;
+
+        long count = logsList.stream()
+                .count();
+
+        String logMessage = logsList.get(0).getFormattedMessage();
+        Level logLevel = logsList.get(0).getLevel();
+        assertEquals("Total repair success in node till now is:1", logMessage);
+        assertEquals(Level.DEBUG, logLevel);
+        assertEquals(1, count);
+    }
+
+    @Test
+    public void testLogForRepairedRatioStatusLogs()
+    {
+        TableReference tableReference = tableReference(TEST_KEYSPACE, TEST_TABLE1);
+        myTableRepairMetricsImpl.repairState(tableReference, 1, 0);
+        // Call the method to be tested
+        StatusLogger.log(myMeterRegistry);
+
+        List<ILoggingEvent> logsList = listAppender.list;
+
+        long count = logsList.stream()
+                .count();
+
+        String logMessage = logsList.get(0).getFormattedMessage();
+        Level logLevel = logsList.get(0).getLevel();
+        assertEquals("Node Repair Ratio is :: 1.0", logMessage);
+        assertEquals(Level.DEBUG, logLevel);
+        assertEquals(1, count);
+
+    }
+
+@Test
+public void testLogForLastRepairedAtStatusLogs()
+    {
+        TableReference tableReference = tableReference(TEST_KEYSPACE, TEST_TABLE1);
+        TableReference tableReference2 = tableReference(TEST_KEYSPACE, TEST_TABLE2);
+        long timeNow = System.currentTimeMillis();
+
+        long timeDiff = 1000L;
+        long expectedLastRepaired = timeNow - timeDiff;
+        long timeDiff2 = 5000L;
+        long expectedLastRepaired2 = timeNow - timeDiff2;
+
+        myTableRepairMetricsImpl.lastRepairedAt(tableReference, expectedLastRepaired);
+        myTableRepairMetricsImpl.lastRepairedAt(tableReference2, expectedLastRepaired2);
+        StatusLogger.log(myMeterRegistry);
+        List<ILoggingEvent> logsList = listAppender.list;
+
+        long count = logsList.stream()
+                .count();
+
+        String logMessage = logsList.get(0).getFormattedMessage();
+        Level logLevel = logsList.get(0).getLevel();
+        assertEquals("Node last repaired at",logMessage.substring(0, 21));
+        assertEquals(Level.DEBUG, logLevel);
+        assertEquals(1, count);
+    }
+
+    @Test
+    public void testLogForNodeRemainingRepairTimeStatusLogs()
+    {
+        TableReference tableReference = tableReference(TEST_KEYSPACE, TEST_TABLE1);
```

Comment:
Would name this as tableReference1, to keep the consistency (as for the rest of this test method)